recipe/meta.yaml

```diff
@@ -387,6 +390,8 @@ outputs:
         - pip
 
   - name: {{ name }}-with-ldap
+    build:
+      skip: true
```

Comment:
This might be okay for testing but there is no reason to skip it for `linux_64` and `osx_64`, where all dependencies are available.  See:
https://github.com/conda-forge/airflow-feedstock/pull/102/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR388

Those comments aren't optional -- they change the behavior depending on the system that the template is being used on.

Reply:
Understood, amended that.